src/app/pages/admin/buyer-list/buyer-list.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component } from '@angular/core';
+import { AppService } from 'app/app.service';
+import { Buyer } from 'app/app.model';
+
+@Component({
+  selector: 'wm-buyer-list',
+  templateUrl: './buyer-list.component.html',
+  styleUrls: ['./buyer-list.component.scss']
+})
+export class BuyerListComponent {
+  public searchText: string;
+  constructor(public appServ: AppService) {}
+
+  /**
+   * @description approve a buyer
+   * @param buyer the buyer to approve
+   */
+  public approveBuyer(buyer: Buyer) {
+    this.appServ.approveUser(buyer.id).subscribe(
+      response => {
+        buyer.approved = true;
+        this.appServ.openSnackBar(`Buyer approved successfully`, 'DISMISS');
+      },
+      () => {
+        this.appServ.openSnackBar(`Oops! Could not approve buyer`, 'DISMISS');
+      }
+    );
+  }
+
+  public denyBuyer(buyer: Buyer) {
+    /**
+     * TODO: Perform operation for denying buyer if distinct from approval flow
```

Comment:
Since we dont have denial flow, We can do a delete call(`/users/{id}`) of the user. and rename this function name as 'deleteUser' for now?

Reply:
Correct. Will do so!